### DISCUSSION TITLE
Scrypted Docker Container Issues with Synology

### DISCUSSION THREAD

**Discussion by cubbieblue16** [koush/scrypted#1384] [Q&A]
**Describe the bug**
I have installed Scrypted on a Synology NAS that is running OS DSM 7.2-64570 Update 3.

I had no issues installing Scrypted, everything went smooth and adding cameras and getting them to run worked just fine.  However, roughly, every 23-24 hours I will receive an email notification from Synology NAS saying that my NAS is offline.  I go into the NAS and the NAS is in face not offline. However, I notice that all my cameras are offline.

Naturally, I went in to investigate and I am not seeing anything in the logs.  So I restarted the container and boom, email comes in that my NAS is back online and the cameras magically start working again.

This happens to me EVERY 23-24 hours, like clock work.  It has been happening since day 1

**To Reproduce**
There is no way to reproduce the error other than waiting 23-24 hours and receiving the email.

**Server (please complete the following information):**
 - OS: Synology NAS that is running OS DSM 7.2-64570 Update 3.
 - Installation Method: Portainer

**Hardware Model (please complete the following information):**
 - Device: Synology SAN

**Client (please complete the following information, if applicable):**
 - Software: Home App

Additional information:

Currently the only solution to this is to create a reoccuring schedule reboot of the docker container.

**Comment by cubbieblue16** [comment]
@koush What more do you need from me to look into this issue? This is super impactful.

**Reply by exponential-2** [reply]
Hi, I have the same issue.
